src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlDocumentWrapperFixture.java

```diff
@@ -0,0 +1,47 @@
+package edu.cornell.kfs.rass.batch.xml.fixture;
+
+import java.util.Date;
+import java.util.List;
+
+import org.joda.time.format.DateTimeFormatter;
+
+import edu.cornell.kfs.fp.batch.xml.fixture.AccountingXmlDocumentFixtureUtils;
+import edu.cornell.kfs.rass.batch.xml.RassXmlDocumentWrapper;
+import edu.cornell.kfs.rass.batch.xml.RassXmlDocumentWrapperMarshalTest;
+
+public enum RassXmlDocumentWrapperFixture {
+        RASS_EXAMPLE("2019-03-15T22:15:07.273", awardFixtures(RassXmlAwardEntryFixture.FIRST, RassXmlAwardEntryFixture.ANOTHER, RassXmlAwardEntryFixture.NULL_AMOUNTS), 
+                agencyFixtures(RassXmlAgencyEntryFixture.SOME, RassXmlAgencyEntryFixture.DoS, RassXmlAgencyEntryFixture.TEST));
+    
+    public final Date extractDate;
```

Comment:
This enum could also benefit from the date parsing/storing enhancements that were mentioned in the award fixture comments.

Reply:
Changed